docs/3.main-concepts/7.humanhandoff.mdx

```diff
@@ -68,8 +79,6 @@ export default class extends React.Component {
     }
   }
 }
-
+```
 > #### NOTE:
-> Both features are only available when the bot is deployed. It's not supported in `botonic serve`
-
-```
\ No newline at end of file
+> Both features are only available when the bot is deployed. You can only emulate the behaviour of `humanHandoff` with the `botonic serve` command.
```

Comment:
imho, It's not supported in `botonic serve` is more supported than mentioning that it can be emulated

Reply:
replaced for: `When you are in development mode (`botonic serve` command), you can press the button 'Continue' in order to emulate the resolution of a case.`